pom.xml

```diff
@@ -126,6 +132,7 @@
     <module>hadoop-shaded-protobuf_3_21</module>
     <module>hadoop-shaded-guava</module>
     <module>hadoop-shaded-avro_1_11</module>
+    <module>hadoop-shaded-jersery</module>
```

Comment:
typo - should be `jersey` - maybe you it should be `jersey_2` to match avro and protobuf